lib/omnibus/packagers/rpm.rb

```diff
@@ -494,7 +495,11 @@ def safe_version
       #   http://rpm.org/ticket/56
       #
       if version =~ /\-/
-        converted = version.gsub('-', '~')
+        if Ohai['platform_family'] == 'wrlinux'
```

Comment:
We should find a way to make this check more general. Should't it also be based on a combination of platform and platform version?